ext/bootsnap/bootsnap.c

```diff
@@ -269,23 +269,25 @@ cache_key_equal(struct bs_cache_key * k1, struct bs_cache_key * k2)
 static VALUE
 bs_rb_fetch(VALUE self, VALUE cachedir_v, VALUE path_v, VALUE handler)
 {
+  VALUE stringified_path_v = rb_get_path(path_v);
```

Comment:
I think we should use the `FilePathValue` macro here to avoid the value to be GCed before we use and give a segfault.

Reply:
@rafaelfranca 

Sorry that was mistake.
Just replaced it with FilePathValue. ( looks much cleaner now 👍)
Thank you really a lot for the review!